packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -6,11 +6,11 @@
         key="new-file-menu-btn-enabled"
         v-oc-tooltip="newButtonTooltip"
         :aria-label="newButtonAriaLabel"
-        variation="inverse"
+        variation="raw"
         :disabled="uploadOrFileCreationBlocked"
         class="oc-background-primary-gradient"
       >
-        <oc-icon name="add" />
+        <oc-icon name="add" color="#ffffff" />
```

Comment:
Please use `variation="inverse"`

Reply:
inverse wont work here because it gets overwritten by the button variation fill type for the icon. Have done it now by using important and custom style.